include/sound/sof/ext_manifest4.h

```diff
@@ -90,6 +90,21 @@ struct sof_man4_segment_desc {
 	uint32_t file_offset;
 } __packed;
 
+/*
+ * bit definition for type in sof_man4_module
+ * uint32_t	load_type : 4;	//MT_BUILTIN, MT_LOADABLE
+ * uint32_t	auto_start : 1;	// single instance created by Module Manager
+ * uint32_t	domain_ll : 1;	// support LL domain
+ * uint32_t	domain_dp : 1;	// support DP domain
+ * uint32_t	lib_code : 1;	// determines if module is place holder for common library code
+ * uint32_t	_rsvd : 24;
+ */
+#define SOF_IPC4_MODULE_LOAD_TYPE(x)		((x) & GENMASK(3, 0))
+#define SOF_IPC4_MODULE_AUTO_START(x)		((x) & BIT(4))
+#define SOF_IPC4_MODULE_DOMAIN_LL(x)		((x) & BIT(5))
+#define SOF_IPC4_MODULE_DOMAIN_DP(x)		((x) & BIT(6))
+#define SOF_IPC4_MODULE_LIB_CODE(x)		((x) & BIT(7))
```

Comment:
These should be (imho):
```c
#define SOF_IPC4_MODULE_LOAD_TYPE	GENMASK(3, 0)
#define SOF_IPC4_MODULE_AUTO_START	BIT(4)
#define SOF_IPC4_MODULE_DP			BIT(6)
#define SOF_IPC4_MODULE_LIB_CODE		BIT(7)
```
and placed around the existing `SOF_IPC4_MODULE_LL`

Reply:
sure